packages/flutter/lib/src/material/ink_decoration.dart

```diff
@@ -356,6 +360,19 @@ class InkDecoration extends InkFeature {
     controller.markNeedsPaint();
   }
 
+  /// Whether the decoration should be painted.
+  ///
+  /// Defaults to true.
+  bool get isVisible => _isVisible ?? true;
+  bool? _isVisible;
```

Comment:
It looks like the fact that _isVisible is nullable is not really important? Maybe simplify to

```suggestion
  bool get isVisible => _isVisible;
  bool _isVisible = true;
```